tardis/base.py

```diff
@@ -26,7 +27,9 @@ def run_tardis(
     virtual_packet_logging : bool
         option to enable virtual packet logging
         [default=False]
-
+    verbose_suppression : bool
+        option to suppress the warnings
+        [default=True]
     Returns
```

Comment:
good succinct explanation of the new flag - after seeing this, perhaps the entire comment on line 10 could be removed

Reply:
Sure will do that